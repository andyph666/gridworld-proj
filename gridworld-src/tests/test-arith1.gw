int a = 33;
int b = 27;
c = a + b;
d = a - b;
e = a * b;
print(c);
print(d);
print(e);