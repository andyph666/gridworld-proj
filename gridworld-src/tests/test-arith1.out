60
6
891